requirements/centos7.requirements.txt

```diff
@@ -1,4 +1,3 @@
-pylint==1.9.5
 astroid==1.6.6
```

Comment:
I forgot to remove `astroid` from the requirements file. I will open another PR to remove them.